src/wasm.h

```diff
@@ -1610,6 +1614,14 @@ class Module {
     }
     importsMap.erase(name);
   }
+
+  void removeImportsWithSubstring(Name name) {
+    for (int i = imports.size() - 1; i >= 0; i--) {
```

Comment:
yeah `std::remove_if` sounds good. But removing from `importsMap` is probably needed too, so maybe you can't just do that if we also need to fish the actual name out. So maybe just a size_t counter from 0 to size(), and then a temp index of size - i or something like that.